srml/staking/src/lib.rs

```diff
@@ -302,6 +314,15 @@ const STAKING_ID: LockIdentifier = *b"staking ";
 /// Counter for the number of eras that have passed.
 pub type EraIndex = u32;
 
+/// Reward of an era.
+#[derive(Encode, Decode, Default)]
+pub struct EraRewards {
+	total: u32,
```

Comment:
Is this assuming that the reward value is always u32? asking bc I have the same dilemma with tip/weight value in some similar fashion.